### PR TITLE
New AJAX Cheat Sheet

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New AJAX Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
Add overview of different terms associated with AJAX

---

Instant Answer Page: https://duck.co/ia/view/ajax_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by sahildua2305** [comment]
@zekiel @moollaza @tagawa Please review this :)

**Comment by sahildua2305** [comment]
@mintsoft Let's merge it if all good?

File: share/goodie/cheat_sheets/json/ajax.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "id": "ajax_cheat_sheet",
+    "name": "AJAX",
+    "description": "Overview of different terms associated with AJAX",
+
+    "metadata": {
+        "sourceName": "DZone",
+        "sourceUrl": "https://dzone.com/refcardz/getting-started-ajax"
+    },
+
+    "aliases": [],
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/ajax.json]: Lets add some aliases

Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/ajax.json]: Or get rid of the `aliases` field if no aliases are necessary 😄

Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/ajax.json]: I will get rid of it. I don't think any alias is required for this one.

@jophab Let me know if you think some alias is required for this cheat sheet. I'll be happy to add that. :)

**Comment by daxtheduck** [comment]
@sahildua2305 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**